site/content/en/docs/Getting started/linux.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Linux"
+linkTitle: "Linux"
+weight: 1
+description: >
+  How to install and start minikube on Linux.
+---
+
+## Installation
+
+{{% tabs %}}
+{{% tab "Direct" %}}
+
+Download and install minikube to /usr/local/bin:
```

Comment:
u can keep this one ! it is important to know it is installing to /usr/local/bin